src/styles/blocks/card.scss

```diff
@@ -0,0 +1,114 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  margin-top: 50px;
```

Comment:
It is forbidden to set sizes and margins for blocks, but for block elements are allowed. The block should stretch to accommodate its children